docs/quickstart/cross-chain-transfers.md

```diff
@@ -0,0 +1,176 @@
+| ***X-Chain*** | [**C-Chain : Import Avax from X-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/evm/buildImportTx-xchain.ts) | _X Chain >> C Chain_ |
+| ***X-Chain*** | [**X-Chain : Export Avax to P-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/buildExportTx-PChain.ts) | _X Chain >> P Chain_ |
+| ***X-Chain*** | [**P-Chain : Import Avax from X-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/platformvm/buildImportTx-XChain.ts) | _X Chain >> P Chain_ |
+| ***P-Chain*** | [**P-Chain : Export Avax to X-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/platformvm/buildExportTx-XChain.ts) | _P Chain >> X Chain_ |
+| ***P-Chain*** | [**X-Chain : Import Avax from P-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/buildImportTx-PChain.ts) | _P Chain >> X Chain_ |
+| ***P-Chain*** | [**P-Chain : Export Avax to C-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/platformvm/buildExportTx-CChain.ts) | _P Chain >> C Chain_ |
+| ***P-Chain*** | [**C-Chain : Import Avax from P-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/evm/buildImportTx-PChain.ts) | _P Chain >> C Chain_ |
+| ***C-Chain*** | [**C-Chain : Export Avax to X-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/evm/buildExportTx-xchain-avax.ts) | _C Chain >> X Chain_ |
+| ***C-Chain*** | [**X-Chain : Import Avax from C-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/buildImportTx-cchain.ts) | _C Chain >> X Chain_ |
+| ***C-Chain*** | [**C-Chain : Export Avax to P-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/evm/buildExportTx-pchain.ts) | _C Chain >> P Chain_ |
+| ***C-Chain*** | [**P-Chain : Import Avax from C-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/platformvm/buildImportTx-CChain.ts) | _C Chain >> P Chain_ |
+
+
+### Step 2 - Modify Your Export
+
+:::info
+
+The script in this tutorial uses **X Chain <-> C Chain** transfers. Depending on which chains you want to interact with, the scripts you use could be different. Please check the table above for more information.
+
+:::
+
+Select the **examples/avm** folder to view the AvalancheJS X-Chain examples. To send AVAX from the X-Chain to the C-Chain, select [```avm/buildExportTx-cchain-avax.ts```](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/buildExportTx-cchain-avax.ts)
+
+![Image for post](/img/ajs-export-cchain-res.png)
+
+#### Modify your Avalanche network configuration
+
+If you are making calls to your Node from another machine, or started your node using [Advanced Configurations](https://docs.avax.network/nodes/build/set-up-node-with-installer#advanced-node-configuration) you might use a custom IP
+```js
+const ip: string = "localhost" | "custom" 
```

Comment:
If the custom IP is an actual IP, then it might be better to put something like
`const ip: string = "localhost" | "<YOUR-NODE-IP>"`
Otherwise people will put the actual "custom" string.

Also, be careful with things like `"localhost" | "custom"`- this is a valid JS code. It does not convey that people just copy/pasting your instruction should use one or the other.

Don't be afraid to waste screen space, scrolling is cheap. You might present it like:

...you might use a local node:
```js
const ip: string = "localhost"
```
Or a custom node IP:
```js
const ip: string = "<YOUR-NODE-IP_HERE>"
```